pkg/kcp/provider/gcp/mock/vpcPeeringStore.go

```diff
@@ -13,32 +14,103 @@ type vpcPeeringEntry struct {
 }
 type vpcPeeringStore struct {
 	m     sync.Mutex
-	items []*vpcPeeringEntry
+	items map[string]*vpcPeeringEntry
 }
 
-func (s *vpcPeeringStore) CreateVpcPeeringConnection(ctx context.Context, name *string, remoteVpc *string, remoteProject *string, importCustomRoutes *bool, kymaProject *string, kymaVpc *string) (*pb.NetworkPeering, error) {
+func getFullNetworkUrl(project, vpc string) string {
+	return fmt.Sprintf("https://www.googleapis.com/compute/v1/projects/%s/global/networks/%s", project, vpc)
+}
+
+func (s *vpcPeeringStore) CreateRemoteVpcPeering(ctx context.Context, remotePeeringName string, remoteVpc string, remoteProject string, importCustomRoutes bool, kymaProject string, kymaVpc string) (*compute.Operation, error) {
```

Comment:
```suggestion
func (s *vpcPeeringStore) CreateRemoteVpcPeering(ctx context.Context, remotePeeringName string, remoteVpc string, remoteProject string, importCustomRoutes bool, kymaProject string, kymaVpc string) (error) {
```
Is operation used anywhere? If not, perhaps we can omit returning it all together